xgettext.js

```diff
@@ -192,13 +193,14 @@ XGettext.prototype._discoverMatches = function( parsedInput ) {
 			var match = {
 				arguments: node.arguments,
 				keyword: functionName,
-				line: node.loc.start.line
+				line: node.loc.start.line,
+				column: node.loc.start.column
 			};
 
 			// Find translator comment
 			_.each( parsedInput.comments, function( translatorComment ) {
-				if ( node.loc.start.line === translatorComment.line.line ||
-					node.loc.start.line - 1 === translatorComment.line.line ) {
+				if ( node.loc.start.line === translatorComment.line ||
```

Comment:
It was working before, but `line` was referring to a line/column position, `comment.loc.start`, hence the need for `line.line` as opposed to just `line`, or something like `loc.line`, so just a minor oddity given the usage of the term `line` elsewhere, so I figured I'd homogenize that. A few lines up, https://github.com/Automattic/xgettext-js/pull/13/commits/50f10eeb41816b3a876f5b128f6e4fdddbc29fb9#diff-0818771692e97ae73428c707b1f158feL149 I changed what `line` was to just be the line number and paired it with a `column` field... although it turned out we never use the column value for comments to it's all a bit moot. Happy to revert those two lines if you'd like.